### ISSUE TITLE
Several MySQL column types are not supported

### ISSUE THREAD

**Issue by Stanpol** [bakwc/mysql_ch_replicator#31]
I have a table:
```sql
--- processing statement:
 CREATE TABLE `test` (
  `id` bigint NOT NULL AUTO_INCREMENT,
  `final` bit(1) NOT NULL,
  `coordinate` point DEFAULT NULL,
  `external_id` binary(16) NOT NULL,
  PRIMARY KEY (`id`),
) ENGINE=InnoDB AUTO_INCREMENT=2906 DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci 
```

This results in exceptions:
```
  File "/.direnv/python-3.11.2/lib/python3.11/site-packages/mysql_ch_replicator/converter.py", line 140, in convert_type
    raise Exception(f'unknown mysql type "{mysql_type}"')
Exception: unknown mysql type "bit(1)"
```
and 
```
File "/.direnv/python-3.11.2/lib/python3.11/site-packages/mysql_ch_replicator/converter.py", line 140, in convert_type
    raise Exception(f'unknown mysql type "{mysql_type}"')
Exception: unknown mysql type "point"
```
and
```
File "/.direnv/python-3.11.2/lib/python3.11/site-packages/mysql_ch_replicator/converter.py", line 140, in convert_type
    raise Exception(f'unknown mysql type "{mysql_type}"')
Exception: unknown mysql type "binary(16)"
```

**Comment by bakwc** [comment]
Added support for `bit(1)` and `point` types in release `0.0.34`. Please upgrade and check if it helped.

**Comment by Stanpol** [comment]
Thank you for the super quick fix, that's very impressive. Will check it shortly. Thanks again.

I also added `binary(16)` to the failing cases.

**Comment by bakwc** [comment]
> I also added binary(16) to the failing cases.

Fixed in `0.0.35`

**Comment by Stanpol** [comment]
All mentioned types are working now. Thank you for your work!
